src/runtime/Object.h

```diff
@@ -1172,6 +1187,14 @@ class Object : public PointerValue {
         return (ObjectRareData*)m_prototype;
     }
 
+    ObjectExtendedExtraData* extendedExtraData()
```

Comment:
It seems that this `extendedExtraData` function is defined but nowhere used.

Reply:
You are right!